src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -87,6 +87,12 @@ const deleteToken = "FDE8472A-6095-4292-ABF7-E35AD435F05F"; // randomly generate
 
 // provides dropdown options for Improvement Notation
 const improvementNotationOptions = [
+  {
+    label: "-",
+    value: "",
+    subtitle: null,
```

Comment:
QDM shouldn't have the improvement notation in QDMMeasureGroups, it is a separate component and should already be having this functionality. Can you double check if this is still necessary, I believe this code can be deleted.